### PR TITLE
fix(web): small issues everywhere

### PR DESCRIPTION
**Pull request by martabal**
This PR:
- removes `start-microservices.sh` and `start-server.sh` usage in favor of `start.sh` (but still keep them)
- fix an issue when creating an album where the "Select Photos" re-appear after entering the album name , reported by @alextran1502 
- when creating an album,  add a margin-top between the title and the description 
- center the text in the chat in the activity viewer
- fix the autoGrowHeight for the text-area in the description
- add a notification when the album / asset description is updated
- fix for #5603

## Screenshots

| Before | After |
| :---: | :---: |
| Screenshot from 2024-02-19 19-36-35 | Screenshot from 2024-02-19 20-41-11 |
| Screenshot from 2024-02-19 19-36-58 | Screenshot from 2024-02-19 19-35-48 |

### CODE REVIEW

**Comment by alextran1502** [comment]
Have you test running `make prod` and see if it works as expected?

**Comment by martabal** [comment]
> Have you tested running `make prod` and see if it works as expected?

Yes, It works fine

**Comment by alextran1502** [comment]
Can we use this opportunity to fix the placeholder text color of "Add a title" in dark mode to be a little brighter?

**Review by danieldietzler — APPROVED** [review]
LGTM

**Review by jrasm91 — APPROVED** [review]
LGTM

**Review by etnoy — APPROVED** [review]

**Comment by alextran1502** [comment]
@jrasm91 This change doesn't affect the user, as I understand, right?

**Comment by jrasm91** [comment]
> @jrasm91 This change doesn't affect the user, as I understand, right?

That is incorrect. Existing installations will have docker compose files with the start command referencing the bash script `start-*.sh`. Those files will no longer exist in the docker image and the containers will fail to come up. It is a breaking change because users are _required_ to update their compose files and change the start commands. This doesn't only apply to docker compose projects, but any references to those custom start commands (of which I believe there are many) will become invalid if we delete the files.

**Comment by alextran1502** [comment]
@jrasm91 The change here is for the `docker-compose.prod.yml` not `docker-compose.yml` though

**Comment by martabal** [comment]
Yes, I don’t think it’s a breaking change

**Comment by etnoy** [comment]
> Yes, I don’t think it’s a breaking change

It is a breaking change since many users still use the old syntax and runs the .sh file directly from their docker compose. Those users need to update their compose file

**Comment by jrasm91** [comment]
> @jrasm91 The change here is for the `docker-compose.prod.yml` not `docker-compose.yml` though

It doesn't matter. The breaking change is the fact that the start-microservices.sh file is being deleted.

File: web/src/lib/components/asset-viewer/asset-viewer.svelte [hunk 1]
```diff
@@ -570,6 +555,8 @@
   };
 </script>
 
+<svelte:window on:keydown={handleKeypress} />
```
Comment (jrasm91) [inline web/src/lib/components/asset-viewer/asset-viewer.svelte:558]: Nice!

File: web/src/routes/(user)/albums/[albumId]/+page.svelte [hunk 2]
```diff
@@ -152,6 +152,19 @@
...
+      albumId = data.album.id;
+      currentAlbumName = data.album.albumName;
+      assetStore = new AssetStore({ albumId: album.id });
+      timelineStore = new AssetStore({ isArchived: false }, album.id);
+      if (album.sharedUsers.length > 0) {
+        getFavorite();
+        getNumberOfComments();
+      }
+      refreshAssetGrid = !refreshAssetGrid;
+    }
```
Comment (jrasm91) [inline web/src/routes/(user)/albums/[albumId]/+page.svelte:167]: This doesn't seem like a "small issue" and I 'm not sure if this actually works. Can you move this into a separate pull request?

Comment (martabal) [reply web/src/routes/(user)/albums/[albumId]/+page.svelte:167]: Aahah sure

Comment (martabal) [reply web/src/routes/(user)/albums/[albumId]/+page.svelte:167]: We use the same trick with [personId]/+page.svelte

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**